supplementary/alica_turtlesim/libalica-turtlesim/src/Teleport.cpp

```diff
@@ -15,11 +15,25 @@ Teleport::Teleport(alica::BehaviourContext& context)
 
 void Teleport::initialiseParameters()
 {
-    // teleport turtle to random place
+    _startTime = std::chrono::high_resolution_clock::now();
+}
+
+void Teleport::run()
+{
+    if (isSuccess()) {
+        return;
+    }
+
+    // Delay so as to avoid rapid success in case of repeated calls
```

Comment:
What is the problem here?  setSuccess() should mean repeated calls return immediately right?

Reply:
In PlaceholderTutorial the MoveSequence plan moves the robot to each of the 4 corners using the IGoTo interface. If Teleport is used here without the delay it will move so fast barely anything is visible. Therefore I put a delay here. I'll improve the comment

Reply:
Perhaps better is to add time as blackboard input to teleport plan?